### PR TITLE
Allow clusterverse to be dynamically acquired via ansible-galaxy as part of the playbook.

### PR DESCRIPTION
**Pull request by dseeley**
+ Updates to allow clusterverse to be dynamically acquired via ansible-galaxy as part of the playbook.
  + Change the EXAMPLE/cluster.yml and redeploy.yml to use include_roles (dynamic load, because clusterverse won't exist yet)
  + Change cli_facts.py plugin from a callback to a vars plugin (callback_plugins are not evaluated in include_role)
  + Remove dependency on ansible facts for epoch time.
+ Simplify the package upgrade logic to use new 'reboot:' action
+ Make the inclusion of 'regionzone' in the inventory dependent on it existing for that cloud type.
+ Add short delay to allow bind9 zone transfers to complete

### CODE REVIEW

File: EXAMPLE/cluster.yml [hunk 1]
```diff
@@ -3,22 +3,28 @@
 - name: Deploy the cluster
   hosts: localhost
   connection: local
-  roles:
-    - { role: clusterverse/clean, tags: [clusterverse_clean], when: clean is defined }
-    - { role: clusterverse/create, tags: [clusterverse_create] }
-    - { role: clusterverse/dynamic_inventory, tags: [clusterverse_dynamic_inventory] }
+  gather_facts: no
+  tasks:
+    - { name: "Get dependent roles via ansible-galaxy", local_action: "command ansible-galaxy install -r requirements.yml", tags: ["always"] }
```
Comment (dipiazzg) [inline EXAMPLE/cluster.yml:8]: Shall we add `--force` after `command ansible-galaxy install -r requirements.yml` to make sure the correct clusterverse role (the one specified in requirements.yml) is imported every time the playbook is run?

Comment (dseeley) [reply EXAMPLE/cluster.yml:8]: Yes, good idea.

Comment (dseeley) [reply EXAMPLE/cluster.yml]: Added https://github.com/sky-uk/clusterverse/pull/70/commits/443f64c3442f658708315b9f7773db6eb7942fb5 to address

**Review by dipiazzg — APPROVED** [review]
LGTM - Successfully tested creation, deletion and re-deploy

**Review by antoineserrano — APPROVED** [review]
Looks good to me. Reviewed with @dipiazzg

**Pull request merged**